lib/client/api.go

```diff
@@ -2751,6 +2751,14 @@ func (tc *TeleportClient) connectToProxy(ctx context.Context) (*ProxyClient, err
 	return pc, nil
 }
 
+// Message for error in connecting to ssh proxy address
+func confirmSSHConnectivityErrorMsg(proxyAddress string) (errorMessage string) {
+	msg := fmt.Sprintf("Unable to successfully connect and authenticate to ssh proxy address=%v. "+
+		"Required for Teleport client. "+
+		"Confirm connectivity and availability.", proxyAddress)
+	return msg
+}
```

Comment:
nit: use godoc style, cut down variables a bit:

```suggestion
// confirmSSHConnectivityErrorMsg returns the error message for Proxy SSH
// connectivity errors.
func confirmSSHConnectivityErrorMsg(proxyAddress string) string {
	return fmt.Sprintf("Unable to successfully connect and authenticate to ssh proxy address=%v. "+
		"Required for Teleport client. "+
		"Confirm connectivity and availability.", proxyAddress)
}
```